### DISCUSSION TITLE
unable to escape : while adding new field name in request Transformer

### DISCUSSION THREAD

**Discussion by sivasubramani300890** [Kong/kong#9729] [Help] [answered]
Hi All,

I am trying to add new field in request Transformer "project:module = Test",but due to : its taking project as field name and remaining as value

Fieldname - project:module
Value - Test

Any suggestion please?

**Comment by StarlightIbuki** [comment, marked as answer]
":" is invalid for HTTP header names and URL parameter names. So, you could URL-encode them, or try to name it in another way.
